### ISSUE TITLE
Backend: secure access connected to database

### ISSUE THREAD

**Issue by veitmo** [amosproj/amos2022ss04-digital-identity#74]
## User story
1. As {Adorsys}
2. I want / need {to be sure that only eligible HR-employees can access DIdentity}
3. So that {no fraudulent action can occur}

## Acceptance criteria
* secure access needs to be connected to database
* when loggin into DIdentity, the credentials are checked against the saved credentials in the database

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by veitmo** [comment]
@Jean28518 
Hi Jean, I have noted down in the last sprint planning that we have to implement the secure access with the authentification process. However, the technical details of this authentication process are not 100% clear to me. Could you maybe have a look at the acceptance criteria and suggest more detailed bullets of what needs to be done?
